[2024-06-03T05:45:39.322+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T05:45:39.351+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T05:45:15.041624+00:00 [queued]>
[2024-06-03T05:45:39.361+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T05:45:15.041624+00:00 [queued]>
[2024-06-03T05:45:39.362+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-03T05:45:39.378+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-03 05:45:15.041624+00:00
[2024-06-03T05:45:39.391+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4213) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T05:45:39.392+0000] {standard_task_runner.py:63} INFO - Started process 4215 to run task
[2024-06-03T05:45:39.391+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'manual__2024-06-03T05:45:15.041624+00:00', '--job-id', '280', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp5lbmzfrb']
[2024-06-03T05:45:39.394+0000] {standard_task_runner.py:91} INFO - Job 280: Subtask load
[2024-06-03T05:45:39.411+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-03T05:45:39.445+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.load manual__2024-06-03T05:45:15.041624+00:00 [running]> on host aa989158b36e
[2024-06-03T05:45:39.540+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T05:45:15.041624+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T05:45:15.041624+00:00'
[2024-06-03T05:45:39.542+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T05:45:39.583+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/google/cloud/firestore_v1/base_collection.py:302 UserWarning: Detected filter using positional arguments. Prefer using the 'filter' keyword argument instead.
[2024-06-03T05:45:45.773+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T05:45:45.774+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T05:45:45.783+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load, run_id=manual__2024-06-03T05:45:15.041624+00:00, execution_date=20240603T054515, start_date=20240603T054539, end_date=20240603T054545
[2024-06-03T05:45:45.816+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T05:45:45.832+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T05:45:45.836+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
